src/sass/components/_products.scss

```diff
@@ -0,0 +1,285 @@
+}
+.products{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding-top: 121px;
+    padding-bottom: 98px;
+
+    &__label{
+        font-family: $font-header;
+        font-size: 18px;
+        line-height: 1.17; 
+        text-align: center;
+        text-transform: uppercase;
+        color: $color-accent;
+    }
+    &__title{
+        font-family: $font-header;
+        font-size: 34px;
+        line-height: 1.15;
+        text-align: center;
+        letter-spacing: 0.04em;
+        text-transform: uppercase;
+        color: $color-highlight;
+        margin: 0;
+    }
+    &__list {
+        display: flex;
+        flex-direction: column;
+    }
+    &__item {
+        width: 100%;
+        height: 485px;
+        border-radius: 24px;
+        padding: 161px 41px 21px 39px;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-evenly;
+        align-items: center;
+        position: relative;
+        margin-top: 180px;
+        &::after {
+            content: url(../images/products/dots.png);
+            position: relative;
+            top: -239px;
+            right: 0;
+          }
+
+        &:nth-child(1) {
+            background: $color-ice-cream-card-bg;
+            &::before {
+                content: url(../images/products/mobile/mobile-ice-cream.png);
+                position: absolute;
+                top: -160px;
+                right: 4%;
+              }
+        }
+        &:nth-child(2) {
+            background: $color-ice-coffee-card-bg;
+            &::before {
+                content: url(../images/products/mobile/mobile-ice-coffee.png);
+                position: absolute;
+                top: -160px;
+                right: 4%;
+              }
+        }
+        &:nth-child(3) {
+            background: $color-milkshakes-card-bg;
+            &::before {
+                content: url(../images/products/mobile/mobile-milkshakes.png);
+                position: absolute;
+                top: -160px;
+                right: 4%;
+              }
+        }   
+    }
+    &__secondary-title{
+        font-family: $font-header;
+        font-size: 20px;
+        line-height: 1.15;
+        text-align: center;
+        letter-spacing: 0.04em;
+        text-transform: uppercase;
+        color: $color-light;
+    }  
+    &__description{
+        font-family: $font-primary;
+        font-weight: bold;
+        font-size: 14px;
+        line-height: 1.5;
+        text-align: center;
+        letter-spacing: 0.04em;
+        color: $color-light;
+        margin-top: 66px;
+        margin-bottom: 30px;
+    } 
+    &__btn{
+        width: 40px;
+        height: auto;
+        padding: 0;
+        border: none;
+        border-radius: 50%;
+        background-image: url(../images/products/arrow.jpg);
+        cursor: pointer;
+    } 
+}
+
+@media screen and (min-width: 480px) and (max-width: 767px){
```

Comment:
min-width chyba w ogóle nie jest potrzebne, wystarczy max-width 767, a jeżeli przy uzyciu min-width, to wtedy 320px, nie 480, chyba, że to jakaś responsywność posrednia?

Reply:
responsywnośc pośrednia to by była chyba jakby szerokości containerów były określane w px a nie %
ogólnie to min-width wydaję mi się, że nie byłoby takim dużym błędem gdyby wartość byłaby chociaż te 320px, bo w takiej szerokości musi wyświetlać się strona poprawnie. Ale i tak lepiej z wersji mobilnej w ogóle wywalić min-width :)

Reply:
Ok zaraz poprawie